thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -264,8 +301,12 @@ def thrust_cb(self, thrust_msg):
         % thruster_interface_path,
     )
     NUM_THRUSTERS = rospy.get_param("/propulsion/thrusters/num", default=8)
-    THRUST_OFFSET = rospy.get_param("/propulsion/thrusters/offset", default=[0, 0, 0, 0, 0, 0, 0, 0])
-    THRUSTER_DIRECTION = rospy.get_param("/propulsion/thrusters/direction", default=[1, 1, 1, 1, 1, 1, 1, 1])
+    THRUST_OFFSET = rospy.get_param(
+        "/propulsion/thrusters/offset", default=[0, 0, 0, 0, 0, 0, 0, 0]
+    )
+    THRUSTER_DIRECTION = rospy.get_param(
+        "/propulsion/thrusters/direction", default=[1, 1, 1, 1, 1, 1, 1, 1]
+    )
```

Comment:
Remove the default param values here - if offset values and directions cannot be loaded in from a config file, the thruster are not going to function as predicted i.e., won't spin in correct directions etc.

Reply:
Done and tested, it works as expected :)